ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml

```diff
@@ -34,4 +34,14 @@ LVLM:
   - level_status
   opt_uses:
   - percentage_sensor
+
+WTS:
```

Comment:
this is called `WDT` in the HVAC namespace, so i would suggest using that name.  we can eventually elevate this concept to the global namespace and reference it in local namespaces, all under the same name.

Reply:
Yes. Committed the changes to add in the global name space

Reply:
Closing the pull request as we will be using HWS_SWTM canonical type.